controllers/ContentExportImport.js

```diff
@@ -3,10 +3,40 @@
 const PLUGIN_ID = 'content-export-import';
 
 const validator = require('./validations');
+const csvHelpers = require("./csvHelpers");
 
 module.exports = {
   importContent: async (ctx) => {
     const importService = strapi.plugins[PLUGIN_ID].services['contentexportimport'];
+
+    let body = ctx.request.body;
+
+    try {
+      // Check if request has files
+      if (ctx.request.files) {
+        // Get files['source']
+        const sourceFile = ctx.request.files["source"].path;
+        const type = ctx.request.files["source"].type;
+
+        // Read and parse file['source']
+        const source = csvHelpers.readFile(
```

Comment:
The `csvHelpers` could be a `fileHelper` or something, otherwise it looks weird to make it parse JSON file as well.